.changeset/itchy-mugs-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/admin.actions.v1": patch
```

Comment:
Since this is a new feature, let's update to minor

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7486/commits/34a926448e3cea96642355bfe473951501c89739